src/ui.text-files/types.ts

```diff
@@ -1,12 +1,14 @@
 import defaultConfig from "./config.ts";
 
+import type { UFileItem } from "../ui.text-file/types.ts";
+
 export type Config = Partial<typeof defaultConfig>;
 
 export interface UFilesProps {
   /**
    * List of file objects.
    */
-  fileList?: Array<object>;
+  fileList?: UFileItem[];
```

Comment:
It should accept array of [File objects](https://developer.mozilla.org/en-US/docs/Web/API/File)